mangaki/mangaki/tests/test_refresh_from_anidb.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+import responses
+from django.conf import settings
+from django.test import TestCase
+
+from django.core.urlresolvers import reverse
+from django.contrib.auth import get_user_model
+from django.contrib import admin
+
+from mangaki.models import Work, Category
+from mangaki.utils.anidb import AniDB, client
+
+
+class RefreshFromAniDBTest(TestCase):
+    @staticmethod
+    def read_fixture(filename):
+        with open(os.path.join(settings.TEST_DATA_DIR, filename), 'r', encoding='utf-8') as f:
+            return f.read()
+
+    @staticmethod
+    def add_to_responses(filename):
+        responses.add(
+            responses.GET,
+            AniDB.BASE_URL,
+            body=RefreshFromAniDBTest.read_fixture('anidb/'+filename),
+            status=200,
+            content_type='application/xml'
+        )
+
+    def setUp(self):
+        self.user = get_user_model().objects.create_superuser(username='test', password='test', email='email@email.email')
+        client.__init__('test', 1)
```

Comment:
Yeah, I'm not totally OK with my solution either ... it was just the only thing that I tried that worked, so I stuck with it ^^

So yeah, I would definitely like to see how to do this properly instead of calling `__init__` ! 👍